src/Ormolu/Printer/Meat/Declaration/Value.hs

```diff
@@ -186,8 +186,7 @@ p_grhs _ (XGRHS NoExt) = notImplemented "XGRHS"
 
 p_stmt :: Stmt GhcPs (LHsExpr GhcPs) -> R ()
 p_stmt = \case
-  LastStmt NoExt _ _ _ ->
-    notImplemented "LastStmt" -- only available after renamer
+  LastStmt NoExt body _ _ -> located body p_hsExpr
```

Comment:
What is the reason you added this? AFAIK `LastStmt` is only available after renamer and so in our cases it'll never be passed.

Reply:
The return value (e.g. `(a, b)` in `[(a, b) | a <- x, b <- y]`) seems to always be `LastStmt`.

Reply:
At the very least, that is what all of my experimentation seems to indicate.

Reply:
If you want to see this fact in action, you can remove the `LastStmt` case.